jpa/eclipselink/src/main/java/org/jipijapa/eclipselink/WildFlyServerPlatform.java

```diff
@@ -31,10 +30,11 @@
 import org.eclipse.persistence.transaction.jboss.JBossTransactionController;
 
 /**
- * The fully qualified name of WildFlyServerPlatform must be set as the value
- * of the eclipselink.target-server property on EclipseLink version 2.3.2 and
- * older. In newer versions where bug https://bugs.eclipse.org/bugs/show_bug.cgi?id=365704
- * has been fixed, setting eclipselink.target-server to "jboss" is sufficient.
+ * The fully qualified name of WildFlyServerPlatform must be set as the value of
+ * the eclipselink.target-server property on EclipseLink version 2.3.2 and
+ * older. In newer versions where bug
+ * https://bugs.eclipse.org/bugs/show_bug.cgi?id=365704 has been fixed, setting
+ * eclipselink.target-server to "jboss" is sufficient.
```

Comment:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=365704 is no longer enough, as setting eclipseLink.target-server to "jboss", will only cover lookup of the TransactionManager, but I don't think that will enable this TransactionSynchronizationRegistry workaround.